### PR TITLE
Create run0.fish

### PR DESCRIPTION
**Pull request by munzirtaha**
Initial run0 completion which could later be improved.

Fixes issue #10657

### CODE REVIEW

File: share/completions/run0.fish [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#
+# Completion for run0
```
Comment (ccoVeille) [inline share/completions/run0.fish:2]: I have no idea what is run0

I would expect this comment to provide a link to the repository and a short explanation about what run0 is about

**Comment by munzirtaha** [comment]
If I had found an explanation on sudo.fish, I would have done it on run0.fish ;)

It's a systemd-based, more secure replacement for sudo with no suid and a fancy red background. I can add
`# See: https://www.freedesktop.org/software/systemd/man/devel/run0.html`
but I think those kind of explanations for inclusion is better suited here than on that file.

**Comment by ccoVeille** [comment]
Thanks for providing the context. It's clearer

Let's maintainers come back to you

**Comment by faho** [comment]
Thanks, merged!

**Pull request merged**